lib/rules/sotd/publish.js

```diff
@@ -0,0 +1,27 @@
+const self = {
+    name: 'sotd.publish'
+,   section: 'document-status'
+,   rule: 'publish'
+};
+
+exports.name = self.name;
+
+exports.check = function (sr, done) {
+    var sotd = sr.getSotDSection();
+    var crType = sr.config.crType;
+    var text = 'This document was published by the (.)+ Working Group as a Candidate Recommendation ' + crType + '.';
```

Comment:
The regex isn't exactly right.
A document could be a joint publication between 2 WGs or published by the "Technical Architecture Group". See https://github.com/w3c/specberus/blob/main/lib/rules/sotd/pp.js#L63

```suggestion
    var text = 'This document was (?:produced|published) by the (.+? Working Group|Technical Architecture Group)( and the (.+? Working Group|Technical Architecture Group))? as a Candidate Recommendation ' + crType + '.';
```